learning/tour-of-beam/frontend/lib/enums.dart

```diff
@@ -0,0 +1 @@
+enum NodeType { group, unit }
```

Comment:
Please list each value on a separate line.
Please run `./gradlew rat` to spot files without the license like this one.